src/program/gc/gc.lua

```diff
@@ -18,10 +19,14 @@ function run (args)
 
    -- Unlink stale snabb resources.
    for _, pid in ipairs(shm.children("//")) do
-      if not syscall.kill(tonumber(pid), 0) then
+      if tonumber(pid) and not syscall.kill(tonumber(pid), 0) then
          shm.unlink("//"..pid)
       end
    end
+
+   -- unmap hugepages
+   memory.cleanup()
```

Comment:
Is this safe?

I expect the `gc` program to only remove "stale" resources that are not in use by any running process. On casual reading though it seems like this could be disruptive? (`spoon` test script would break if there was a GC during startup?)